### DISCUSSION TITLE
Is there a way for URL /docs to be the root address or other name?

### DISCUSSION THREAD

**Discussion by odhs** [colinwilson/lotusdocs#74] [Q&A]
That is, the website appears at the address `/` ? Without needing to add `/docs/`?

When I compile with `hugo -b /`  I have a website where I need to put /docs/ to access the website. 

Or would it also be possible if I could change the name of the folder where the docs content will be, for example: "manual", thus accessing the documentation `/manual/`

Thank you

**Comment by colinwilson** [comment]
It's possible, but a bit of a pain to manage and maintain.

See my answer [here](https://github.com/colinwilson/lotusdocs/discussions/60#discussioncomment-7004316).

I am looking into a better way to provide this option in a future update.
